desktop/hardened-desktop/enhanced-container-isolation/index.md

```diff
@@ -0,0 +1,61 @@
+---
+description: Enhanced Container Isolation - benefits, why use it, how it differs to Docker rootless, who it is for
+keywords: containers, rootless, security, sysbox, runtime
+title: What is Enhanced Container Isolation?
+---
+
+>Note
+>
+>Enhanced Container Isolation is currently in [Early Access](../../release-lifecycle.md#early-access-ea) and available to Docker Business customers only. 
+
+Enhanced Container Isolation provides an additional layer of security that prevents containers from running as root in the Docker Desktop Linux VM. This ensures a strong container-to-host isolation and locks in any security configurations that have been created, for instance through Registry Access Management policies or with Admin Controls. 
+
+### Who is it for?
+
+- For organizations that want to prevent container attacks and reduce vulnerabilities.
+- For organizations that want ensure stronger container isolation that is easy and intuitive to implement on developers' machines.
+
+### What happens when Enhanced Container Isolation is switched on?
+
+When Enhanced Container Isolation is enabled using [Admin Controls](../admin-controls/index.md), developers can no longer:
```

Comment:
This is worded in the negative sense, which appeals to admins but may scare developers. How about rewording it on a positive sense as follows?

"When Enhanced Container Isolation is enabled using Admin Controls, the following features are enabled:

* All user containers are automatically run in Linux User Namespaces (stronger isolation).
* The root user in the container maps to an unprivileged user at VM level.
* Users can continue using containers as usual, including bind-mounting host directories, volumes, networking configurations, etc. 
* Privileged containers work, but they are only privileged within the container's Linux User Namespace (not in the Docker Desktop VM).
* Containers can no longer share namespaces with the Docker Desktop VM (e.g., --network=host, --pid=host).
* Containers can no longer modify configuration files in the Docker Desktop VM.
* Containers become harder to breach (e.g., sensitive system calls are vetted and portions of /proc and /sys are emulated).

All of this is done automatically, without requiring any special user configuration and with minimal impact in performance and developer productivity. For more information on how Enhanced Container Isolation works, see [LINK-TO-SECTION].

Reply:
+1 ! would be great to call out all these positives.